api/Updated-Metrics.md

```diff
@@ -0,0 +1,368 @@
+# Updates for Metrics
+
+This is an update for the existing metrics document, which is being left in place for now as a point of comparison.
+
+## Assumed data
+
+In the following we assume that we have variables of the following form defined:
+
+```python
+y_true = [0, 1, 0, 0, 1, 1, ...]
+y_pred = [1, 0, 0, 1, 0, 1, ...]
+A_sex = [ 'male', 'female', 'female', 'male', ...]
+A_race = [ 'black', 'white', 'hispanic', 'black', ...]
```

Comment:
I find it better when fairness related docs don't include terms such as `sex` or `race` so that they don't get pulled in discussions such as why `sex` and not `gender`, why `male/female` and not `man/woman`, and where are the other genders?

People tend to put place holders such as `race = ['A', 'B', 'C', 'A', ...]` or `sensitive_attr2 = ...`

But it's a side note, feel free to ignore if you think it's not valid.

Reply:
Agreed. Will change.